src/tcpstream.c

```diff
@@ -146,9 +187,23 @@ void *run_ntttcp_sender_tcp_stream(void *ptr)
 				if (sc->domain == AF_INET) {
 					(*(struct sockaddr_in *)&local_addr).sin_family = AF_INET; /* local_addrs[i].ss_family = AF_INET; */
 					(*(struct sockaddr_in *)&local_addr).sin_port = htons(client_port);
+					if(sc->source_addr_bind)
+					{
+						if (inet_pton(AF_INET, sc->source_address,  &((*(struct sockaddr_in *)&local_addr).sin_addr)) <= 0) {
+							ASPRINTF(&log, "Invalid V4 address or Address %s not supported", sc->source_address);
```

Comment:
V4 -> IPv4

Reply:
I think we have validate the IP address before running here. Do we still need this 'if' test?